### PR TITLE
card-alert-components-created

### PR DESCRIPTION
**Pull request by agrawalshanu708**
created card  and alert component

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
good to merge

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@import "../resetcss.css";
+
+  
+  /* ----------------------------------------------------------------------------------------------------------------------- */
+    .alert-container{
+        display: flex;
+        flex-direction: column;
+      /* border: 1px solid black; */
+      margin : 2rem 0 2rem 0;
```
Comment (zabihshaik) [inline alert/alert.css:9]: Can use shorthand here, like margin: 2rem 0; is enough.

File: alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (zabihshaik) [inline alert/alert.html:7]: Better to add meta tag with your name

File: card/cardreset.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+.card-tag {
+    font-size:var(--card-tag-font-size);
+    color: gray;
+    /* border:1px solid black; */
+}
+
+.card-desc {
+    font-size:var(--card-description-font-size);
+    /* border:1px solid black; */
```
Comment (zabihshaik) [inline card/cardreset.css:30]: Can remove comments if not necessary.

**Review by zabihshaik — COMMENTED** [review]
Everything looks Good. Just a few suggestions as Comments.

**Pull request merged**